### DISCUSSION TITLE
rsocketjs + typescript + react cahnnel communication.

### DISCUSSION THREAD

**Discussion by mkrzywanski** [rsocket/rsocket-js#227] [Q&A]
Hello I have  problem with consuming channel from rsocketjs + typescript + react frontend. The sample code is shown shown in this SO question : https://stackoverflow.com/questions/72259018/rsocketjs-channel-not-established-with-spring-boot-rsocket-server. Maybe someone knows the solution?

**Comment by viglucci** [comment]
Hey @mkrzywanski ,

I'm not sure about the versions prior to `1.0.0-alpha.1`, but I was able to get a working proof of concept using the latest alpha versions.

You can take a look here if you are interested: https://github.com/viglucci/rsocket-chat-demo/blob/8a5e2154d9d7be1b23906a27c3a67b4b63610228/frontend/src/App.jsx

The React patterns may not be perfect, but I think this would be a solid place to start from.
